packages/app/source/client/controllers/todos-controller.js

```diff
@@ -20,23 +23,37 @@ Space.Object.extend(Todos, 'TodosController', {
     this.send(new Todos.CreateTodo({
       targetId: this.configuration.todoListId,
       title: event.title,
-      id: new Guid(),
       isCompleted: false
```

Comment:
Was there a technical reason we needed to supply this up front? I know I advocated for this to stay, but maybe it makes more sense to be controlled internally.